src/Views/TryInstallView.vala

```diff
@@ -83,6 +83,10 @@ public class TryInstallView : AbstractInstallerView {
         var next_button = new Gtk.Button.with_label (_("Install %s").printf (Utils.get_pretty_name ()));
         next_button.get_style_context ().add_class (Gtk.STYLE_CLASS_SUGGESTED_ACTION);
 
+        var shutdown_button = new Gtk.Button.with_label (_("Shut Down…"));
```

Comment:
Make sure to actually connect this to the shutdown session action is in: https://github.com/elementary/installer/blob/master/src/Views/SuccessView.vala#L68